tests/services/components/test_access_component.py

```diff
@@ -17,7 +17,7 @@
 
 from invenio_rdm_records.proxies import current_rdm_records
 from invenio_rdm_records.records import RDMRecord
-from invenio_rdm_records.services.components import AccessComponent
+from invenio_rdm_records.services.components.access import AccessComponent
```

Comment:
minor: since its exported in `__init__.py` you can do:
```suggestion
from invenio_rdm_records.services.components import AccessComponent
```